src/CheckOut.js

```diff
@@ -218,6 +218,7 @@ class CheckOut extends React.Component {
         itemBarcode: PropTypes.string.isRequired,
       }),
     }),
+    intl: PropTypes.object.isRequired,
```

Comment:
Look like we can remove it

Reply:
Thank you for the observation. It's removed now